src/test/java/JiraTester.java

```diff
@@ -102,8 +102,8 @@ public static void main(String[] args) throws Exception {
 
     private static void callUniq(final JiraRestService restService) throws Exception {
         long start = System.currentTimeMillis();
-        List<Issue> issues =
-                restService.getIssuesFromJqlSearch("key in ('JENKINS-53320','JENKINS-51057')", JiraSession.MAX_ISSUES);
+        List<Issue> issues = restService.getIssuesFromJqlSearch(
+                "key in ('JENKINS-53320','JENKINS-51057')", 100);
```

Comment:
this was testing MAX_ISSUES, not the default, so use the `MAX_ALLOWED_ISSUES_FROM_JQL` that you introduced